src/components/Nav/Nav.scss

```diff
@@ -68,65 +71,114 @@
     color: $unselectedHoverForegroundColor;
     background: $unselectedHoverBackgroundColor;
   }
+
+  &.ms-Nav-chevronButton--group {
+    width: 100%;
+    height: $navnode-height;
+    border-bottom: 1px solid $dividerColor;
+  }
 }
 
-.ms-Nav-groupChevron.ms-Icon {
+.ms-Nav-chevron.ms-Icon {
   position: absolute;
-  @include left(15px);
-  height: 40px;
-  line-height: 40px;
-  font-size: 15px;
+  @include left(8px);
+  height: $navnode-height;
+  line-height: $navnode-height;
+  font-size: 12px;
   transition: transform .1s linear;
 }
 
-.ms-Nav-group.is-expanded .ms-Nav-groupChevron {
+.ms-Nav-group.is-expanded .ms-Nav-chevronButton--group .ms-Nav-chevron  {
   transform: rotate(-180deg);
 }
 
-.ms-Nav-link {
+.ms-Nav-linkText {
+  vertical-align: middle;
+}
+
+.ms-Nav-compositeLink {
   display: block;
   position: relative;
-  height: 40px;
-  line-height: 40px;
-  text-decoration: none;
-  padding: 0 20px;
-  cursor: pointer;
-  text-overflow: ellipsis;
-  text-decoration: none;
-  white-space: nowrap;
-  overflow-x: hidden;
 
+  .ms-Nav-chevronButton.ms-Nav-chevronButton--link {
+    display: block;
+    width: 28px;
+    height: $navnode-height;
+    position: absolute;
+    top: 0;
+    @include left(0);
+    z-index: 1;
+    padding: 0;
+    margin: 0;
+
+    .ms-Nav-chevron {
+      position: relative;
+    }
+  }
+
+  & a {
+   color: $unselectedForegroundColor;
+  }
   color: $unselectedForegroundColor;
   background: $unselectedBackgroundColor;
 
   &:hover {
-    color: $unselectedHoverForegroundColor;
-    background: $unselectedHoverBackgroundColor;
-  }
 
-  &.is-selected {
-    position: relative;
-    color: $selectedForegroundColor;
-    background: $selectedBackgroundColor;
+     & a,
+     & .ms-Nav-chevronButton {
+       background: $unselectedHoverBackgroundColor;
+       color: $unselectedHoverForegroundColor;
+     }
+  }
 
-    &:hover {
-      color: $selectedHoverForegroundColor;
-      background: $selectedHoverBackgroundColor;
+  &.is-expanded {
+    .ms-Nav-chevron {
+      transform: rotate(-180deg);
     }
+  }
 
-    &:after {
-      @include border-left(2px, solid, $selectedForegroundColor);
-
-      content: '';
-      position: absolute;
-      top: 0;
-      right: 0;
-      bottom: 0;
-      left: 0;
+  &.is-selected {
+    & a,
+    & .ms-Nav-chevronButton {
+
+      color: $ms-color-themePrimary;
+      background: $selectedBackgroundColor;
+
+      &:after {
+        @include border-left(2px, solid, $selectedForegroundColor);
+
+        content: '';
+        position: absolute;
+        top: 0;
+        @include right(0);
+        bottom: 0;
+        @include left(0);
+      }
     }
   }
 }
 
-.ms-Nav-groupButton, .ms-Nav-link {
+.ms-Nav-link {
+  display: block;
+  position: relative;
+  height: $navnode-height;
+  line-height: $navnode-height;
+  text-decoration: none;
+  padding: 0 20px;
+  cursor: pointer;
```

Comment:
Isn't it already a pointer? To me adding a cursor typically suggests something that may not be accessible.